help/marking-messages-as-read.md

```diff
@@ -9,25 +9,10 @@ configure whether messages are marked as read automatically when you scroll.
 
 ## Configure whether messages are automatically marked as read
 
-{start_tabs}
-
-{tab|mobile}
-
-{!mobile-profile-menu.md!}
-
-1. Tap **Settings**.
-
-1. Tap **Mark messages as read on scroll**.
-
-1. Select **Always**, **Never** or **Only in conversation views**.
-
-{end_tabs}
-
-## Do not mark messages as read
-
-You may want to configure the Zulip mobile app to never mark messages as read
-automatically. For example, this can be useful if you often want to follow up
-on messages when you are at your computer. You will still be able to
+You may want to configure the Zulip mobile app to automatically mark messages
+as read or not, depending on your preferences. For example, setting the app to
+never mark messages as read can be useful if you tend to follow up on the
+messages when you are on your computer. You will still be able to
```

Comment:
How about something like the following for the intro:

```suggestion
You may want to configure whether the Zulip mobile app will automatically
mark messages as read. For example, it can be useful to set the app to never
mark messages as read  if you often want to follow up on messages when you
are at your computer. You will still be able to
```

Reply:
Hmm, yeah I like it, it flows better than what I had. Updating the PR.

Reply:
Actually would it be better if we made it "whether the Zulip mobile app will automatically mark messages as read or not." just to make it a bit more clear?

Reply:
That sounds good to me!